internal/rbd/rbd_util.go

```diff
@@ -200,6 +200,30 @@ func createImage(ctx context.Context, pOpts *rbdVolume, cr *util.Credentials) er
 	return nil
 }
 
+// Open the rbdVolume after it has been connected.
+func (rv *rbdVolume) open() (*librbd.Image, error) {
+	if rv.RbdImageName == "" {
+		var vi util.CSIIdentifier
+		err := vi.DecomposeCSIID(rv.VolID)
```

Comment:
why this is required here? the RbdImageName won't be empty as it's retrieved from the omap.

Reply:
I'm not sure where it came from. It probably was added moths ago, but since then the fetching of the name has changed. I'll drop it and see if things keep working as expected.